src/vs/workbench/services/editor/common/editorOverrideService.ts

```diff
@@ -61,6 +60,8 @@ export interface IEditorType {
 	readonly providerDisplayName: string;
 }
 
+export type UntypedEditorInput = IResourceEditorInput | IUntitledTextResourceEditorInput | IResourceDiffEditorInput;
```

Comment:
Should this be a type exported from where `IResourceEditorInput` is also exported?

Reply:
There was an existing type which I switched to using. `IResourceEditorInputType`